port/cpl_vsil_libarchive.cpp

```diff
@@ -441,7 +477,7 @@ class VSILibArchiveFilesystemHandler final : public VSIArchiveFilesystemHandler
     }
     virtual std::vector<CPLString> GetExtensions() override
     {
-        return {".7z", ".lpk", ".lpkx", ".mpk", ".mpkx"};
+        return {".7z", ".lpk", ".lpkx", ".mpk", ".mpkx", ".rar"};
```

Comment:
the returned extensions should probably depend on the prefix

Reply:
Thanks. I'll make that change.